types_linq/util.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+from collections.abc import MutableMapping, MutableSet
+from typing import Tuple, Iterator, Hashable, List, Dict, Iterable, Set, Optional
+
+from .more_typing import (
+    TKey,
+    TValue
+)
+
+class comparableMap(MutableMapping[TKey, TValue]):
```

Comment:
I don't see a reason currently for having this empty class. lets just remove it to reduce lines